### PR TITLE
update to 1.8.3

### PR DESCRIPTION
**Pull request by XPXPv2**

### CODE REVIEW

**Comment by eli-schwartz** [comment]
#17 ?

You don't need to close the PR and make a new one in order to address feedback, just update the PR next time. ;)

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,29 @@
+project('spdlog', 'cpp', version : '1.8.2', license: 'MIT',
```
Comment (eli-schwartz) [inline meson.build:1]: This regressed from #17 which did mark it as 1.8.3

```diff
$ git --no-pager diff --irreversible-delete refs/pull/17 refs/pull/18
diff --git a/meson.build b/meson.build
index 8d68c5f..aacafb3 100644
--- a/meson.build
+++ b/meson.build
@@ -1,4 +1,4 @@
-project('spdlog', 'cpp', version : '1.8.3', license: 'MIT',
+project('spdlog', 'cpp', version : '1.8.2', license: 'MIT',
         default_options : ['cpp_std=c++11'])
 
 inc = include_directories('include')
@@ -10,7 +10,7 @@ spdlog_compile_args = []
 
 
 if get_option('external_fmt')
-    fmt_dep = dependency('fmt',fallback:['fmt','fmt_dep'])
+    fmt_dep = dependency('fmt')
     spdlog_dependencies += fmt_dep
     spdlog_compile_args += '-DSPDLOG_FMT_EXTERNAL'
 endif
diff --git a/subprojects/fmt.wrap b/subprojects/fmt.wrap
deleted file mode 100644
index 8ce37d9..0000000
```

**Review by eli-schwartz — CHANGES_REQUESTED** [review]

File: upstream.wrap [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = spdlog-1.8.3
+
+source_url = https://github.com/gabime/spdlog/archive/refs/tags/v1.8.3.tar.gz
```
Comment (eli-schwartz) [inline upstream.wrap:4]: What's the intention of refs/tags/ here? It works with

```diff
-source_url = https://github.com/gabime/spdlog/archive/refs/tags/v1.8.3.tar.gz
+source_url = https://github.com/gabime/spdlog/archive/v1.8.3.tar.gz
```

And typically I see the latter.

**Comment by eli-schwartz** [comment]
And 4 hours after #17 got opened, upstream released both 1.8.4 and 1.8.5 so that's fun. Can we go straight to that?

Given 1.8.5 is a bugfix for 1.8.3 to "Fix compiling with C++17", I'm figuring it's probably safer to *not* have this one in the wrapdb. It was out for less than a day before being superseded, anyway...

**Comment by XPXPv2** [comment]
I make a new pull request as it was just easier to delete and remake the old 1.8.3 branch

**Pull request closed without merging**